### PR TITLE
Tiny fixes of Layout, Box and FileHandler 

### PR DESCRIPTION
**Pull request by matzimowski**
closes https://github.com/brainly/style-guide/issues/1869
closes https://github.com/brainly/style-guide/issues/1886
closes https://github.com/brainly/style-guide/issues/1872

thanks @seahindeniz for reporting these issues

### CODE REVIEW

File: src/components/box/Box.jsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ const Box = ({
     {
       [`sg-box--${String(color)}`]: color,
       [`sg-box--padding-${String(padding)}`]: padding !== null && padding,
-      [`sg-box--border-color-${String(borderColor)}`]: borderColor,
+      [`sg-box--border-color-${String(borderColor)}`]: border && borderColor,
```
Comment (matzimowski) [inline src/components/box/Box.jsx:134]: https://github.com/brainly/style-guide/issues/1869

File: src/components/file-handler/FileHandler.jsx [hunk 2]
```diff
@@ -110,10 +110,14 @@ const FileHandler = ({
   className,
   ...props
 }: PropsType) => {
-  const fileHandlerClass = classNames('sg-file-handler', {
-    'sg-file-handler--closable': onClose,
-    [`sg-file-handler--${COLORS_MAP[color]}`]: color,
-  });
+  const fileHandlerClass = classNames(
```
Comment (matzimowski) [inline src/components/file-handler/FileHandler.jsx:113]: https://github.com/brainly/style-guide/issues/1886

File: src/components/layout/Layout.jsx [hunk 3]
```diff
@@ -13,7 +13,6 @@ type PropsType = {
   noMarginTop?: boolean,
   reversedOrder?: boolean,
   fullPage?: boolean,
-  wide?: boolean,
```
Comment (matzimowski) [inline src/components/layout/Layout.jsx:16]: https://github.com/brainly/style-guide/issues/1872

**Review by coderitual — APPROVED** [review]
👏

**Review by ElaLukaszewicz — APPROVED** [review]

**Pull request merged**